### ISSUE TITLE
A single slash path is added to non-HTTP URLs

### ISSUE THREAD

**Issue by bradleyayers** [medialize/URI.js#221]
Take this example:

    new URI("a://b.c").absoluteTo("").toString()
    "a://b.c/"

The slash should only be added for _some_ schemes, e.g. HTTP.

**Comment by rodneyrehm** [comment]
true, we could try limiting this behavior to the [schemes we know the ports of](https://github.com/medialize/URI.js/blob/gh-pages/src/URI.js#L223-L230).

**Comment by rodneyrehm** [comment]
oh, wait, I overlooked the `absoluteTo()` call. what do you expect to happen there that makes `/` become a problem?

**Comment by bradleyayers** [comment]
I stumbled on this because I was writing some tests cases that compared the way the browser resolves URLs (against `window.location.href`), compared with the way URI.js does it.

For the browser I used the code:

    /**
     * Make a URL absolute to the current document location.
     *
     * @param {string} url A URL to make absolute relative to the current window location.
     * @returns {string}
     *
     * @example
     *   > window.location.href
     *   "http://example.com/bar?baz"
     *   > resolveUrl("/foo")
     *   "http://example.com/foo"
     *   > resolveUrl("//example2.pl/bar")
     *   "http://example2.pl/bar"
     */
    function toAbsoluteUrl(url) {
        var a = window.document.createElement("a");
        a.href = url;
        return a.href;
    }

So I was just surprised when the browser behaved differently to URI.js. The behaviour for http and https schemes were the same as the browser, but for non-standard schemes (i.e. random strings) the behaviour was different.
